src/mongocxx/test/spec/unified_tests/runner.cpp

```diff
@@ -45,8 +45,8 @@ using bsoncxx::builder::basic::kvp;
 using bsoncxx::builder::basic::make_document;
 
 using schema_versions_t =
-    std::array<std::array<int, 3 /* major.minor.patch */>, 1 /* supported version */>;
-constexpr schema_versions_t schema_versions{{{{1, 1, 0}}}};
+    std::array<std::array<int, 3 /* major.minor.patch */>, 2 /* supported version */>;
+constexpr schema_versions_t schema_versions{{{{1, 1, 0}}, {{1, 5, 0}}}};
```

Comment:
Can you add a comment noting that this only partially supports 1.5.0?

It is warranted here since these tests are important to implement now, and do not require full 1.5.0 support. But this may be surprising later when the test runner fails to run other tests in unified test format 1.2.0-1.5.0.